src/reducers/index.js

```diff
@@ -1,7 +1,15 @@
-import { DELETE_CARD, TOGGLE_ADD_MODAL, ADD_CARD } from '../actions';
+import {
+  DELETE_CARD,
+  TOGGLE_ADD_MODAL,
+  ADD_CARD,
+  SHOW_EDIT_MODAL,
+  HIDE_EDIT_MODAL,
+  EDIT_CARD
+} from '../actions';
 
 const initialState = {
   showNewTaskModal: false,
```

Comment:
Also it looks like this property isn't being used anywhere.

Reply:
being used to toggle the newTaskModal on line 58?

Reply:
Touche.